use-timescale/compression/index.md

```diff
@@ -19,7 +19,7 @@ hypertables.
 <UsageBasedStorage />
 
 <Highlight type="warning">
-Compression alters data on your disk, so always back up before you start.
+For databases in production, first test compression on a fork.
 </Highlight>
 
```

Comment:
@Rajakavitha1 Let's just remove this entire warning. We expect this operation to be very safe and this just scares people away and prevents them from enabling compression.

Reply:
I've just committed the change and will approve the PR.